blockchain/processing.py

```diff
@@ -606,9 +626,85 @@ def _execute_phase_4(self, config, phase_3_info):
 
             print "phase 4 executed"
 
-    def _execute_phase_5(self, config, phase_5_info):
+    def _execute_phase_5(self, config, verification):
         """ public, Bitcoin bridge phase """
-        print "phase 5 executed"
+        phase = 5
+        verification_record = verification['record']
+
+        verification_info = verification['verification_info']
+        verification_record['verification_info'] = verification_info
+
+        if validate_verification_record(verification_record, verification_info):
+            timestamp_db.insert_verification(verification_record)
+            verification_record['block_id'] = None
```

Comment:
We set them equal to None because I think the reasoning was that the records can be sent from any of the phases to the phase 5 node and so it doesn't have to be dependent on which phase is sent up to the public broadcast.